docs/guide/configuration.md

```diff
@@ -134,13 +137,15 @@ agent = NextGenUIAgent(config=config)
 
 ```python
 from next_gen_ui_agent import NextGenUIAgent
+from next_gen_ui_agent.agent_config import parse_config_yaml
 
 yaml_config = """
 component_system: json
 component_selection_strategy: two_llm_calls
 unsupported_components: true
 """
 
-# Pass YAML string directly to constructor
-agent = NextGenUIAgent(config=yaml_config)
+# Pass YAML string directly
+config = read_config_yaml_file(yaml_config)
```

Comment:
`NextGenUIAgent` is still able to accept `config` string with on yaml, so maybe keep this example as it was before? Or did you want to remove this capability?

Reply:
you're right. reverting back.